trade-finance-manager-ui/server/controllers/case/activity/index.js

```diff
@@ -8,19 +8,25 @@ const { DEAL } = CONSTANTS;
 
 const MAX_COMMENT_LENGTH = 1000;
 
-const mappedActivities = (activities) => activities.map((activity) => ({
-  label: {
-    text: activity.label,
-  },
-  text: activity.text,
-  datetime: {
-    timestamp: fromUnixTime(new Date(activity.timestamp)),
-    type: 'datetime',
-  },
-  byline: {
-    text: `${activity.author.firstName} ${activity.author.lastName}`,
-  },
-}));
+const mappedActivities = (activities) => {
+  if (!activities) {
+    return false;
```

Comment:
Would an empty array be better here? Since if there is data it returns an array

Reply:
Would need to return false to keep it consistent with other function.